### ISSUE TITLE
When adding album(s) to a playlist, use the current visible order

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#691]
Original [issue 691](https://code.google.com/p/quodlibet/issues/detail?id=691) created by pevzi23 on 2011-03-15T19:01:38.000Z:

When selecting several albums with ctrl key the tracks in playlist are getting sorted immediately. It would be nice if they were appended to tracklist in the order the albums were selected. Is it possible to implement such a feature?

**Comment by bassstorm** [comment]
Oh yes, that's really looks weird for casual users (including me).
I think the problem lies in mixing of playlist and browser concepts. Where browser is actually for... browsing, but playlist for organizing your findings in a fully custom way.
Though, I still didn't get use to it, there's nothing to fix or improve. Maybe documentation :)

@CreamyCookie kindly have a look as well, thanks!

**Comment by CreamyCookie** [comment]
@bassstorm Hmm.. maybe I misunderstood, but it should be possible to use the current ordering from albums when adding tracks to a playlist.

**Comment by bassstorm** [comment]
@CreamyCookie yes, but OP means current Album view tracklist. What I mean is just that it works as designed.

**Comment by CreamyCookie** [comment]
@bassstorm Ah.. I see. I guess we could also do that. (Somehow store the order of album selected.) Would be a nice feature with basically no disadvantage.

> What I mean is just that it works as designed.

Yeah, definitely works as designed, but this is an enhancement report, so that's fine.
